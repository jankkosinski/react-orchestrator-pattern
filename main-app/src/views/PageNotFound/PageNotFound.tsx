import React from 'react';
import './styles.css';

export const PageNotFound = () => {
  return <div className='container'>Page Not Found</div>;
};
