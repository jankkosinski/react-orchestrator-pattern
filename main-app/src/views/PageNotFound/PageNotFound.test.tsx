import { render } from '@testing-library/react';
import React from 'react';
import { PageNotFound } from './PageNotFound';

describe('PageNotFound', () => {
  it('Renders the PageNotFound component', () => {
    const { container } = render(<PageNotFound />);
    expect(container).toMatchSnapshot();
  });
});
