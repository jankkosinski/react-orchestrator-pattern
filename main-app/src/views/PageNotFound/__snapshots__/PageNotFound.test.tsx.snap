// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`PageNotFound > Renders the PageNotFound component 1`] = `
<div>
  <div
    class="container"
  >
    PageNotFound
  </div>
</div>
`;
